sources/driverdog/src/main.rs

```diff
@@ -89,16 +102,32 @@ struct DriverConfig {
     object_files: HashMap<String, Linkable>,
 }
 
+/// Holds the configuration used to copy kernel modules
+#[derive(Deserialize, Debug)]
+struct CopyingDriverConfig {
+    #[serde(rename(deserialize = "lib-modules-path"))]
```

Comment:
nit: I think almost all of these can be replaced with [`#[serde(rename_all = kebab-case)]`](https://serde.rs/container-attrs.html#rename_all) on the container. I realize you are just following existing style, though.